platform/src/test/kotlin/org/stellar/anchor/platform/callback/PlatformIntegrationHelperTest.kt

```diff
@@ -42,7 +43,16 @@ class PlatformIntegrationHelperTest {
             (currentTimeMilliseconds + JWT_EXPIRATION_MILLISECONDS) / 1000L
           )
 
-        val jwtService = JwtService(null, null, null, null, "secret", "secret", "secret")
+        val jwtService =
+          JwtService(
+            null,
+            null,
+            null,
+            null,
+            "ssecretsecretsecretsecretecretsecretsecretsecret".toSecretKey(),
```

Comment:
💡 Is `ssecretsecretsecretsecretecretsecretsecretsecret` necessary? 😄

Reply:
Yes, it needs to be 256 bit